woocommerce/sensei-media-attachments
# Change text domain to match plugin slug

As required by translate.wordpress.org

### Changes proposed in this Pull Request

* Change text domain from `sensei_media_attachments` to `sensei-media-attachments`
* There was also a typo in the filenames for the `sl_SI` translation.

### Testing instructions

* Ensure translations are loaded for the current languages.
* Ensure `gulp pot` generates the pot file correctly.
